examples/nasjob-example-envelopenet.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
+metadata:
+  namespace: kubeflow
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: nasenvelopenet
+spec:
+  studyName: nasenvelopenet
+  owner: crd
+  optimizationtype: maximize
+  objectivevaluename: MeanSSS
+  optimizationgoal: 0.99
+  requestcount: 6
+  metricsnames:
+    - accuracy
+  nasConfig:
+    graphConfig:
+      numLayers: 3
+      inputSize:
+        - 32
+        - 32
+        - 3
+      outputSize:
+        - 10
+      max_layers_per_stage:
```

Comment:
You can't add this parameter in `graphConfig`

Reply:
I know, changed